### PR TITLE
K8SPXC-927: add `serviceLabels` and `serviceAnnotations` for PXC

### PR DESCRIPTION
**Pull request by pooknull**
[K8SPXC-927](https://jira.percona.com/browse/K8SPXC-927) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

https://jira.percona.com/browse/K8SPXC-927

**DESCRIPTION**
---
**Problem:**
*`serviceLabel` and `serviceAnnotation` are available to `haproxy` and `proxysql` sections, but not for the `pxc`.*

**Solution:**
*Add `serviceLabel` and `serviceAnnotation` fields to the `pxc` section.*

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [x] Is an E2E test/test case added for the new feature/change?
- [x] Are unit tests added where appropriate?
- [x] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [x] Are all needed new/changed options added to default YAML files?
- [x] Are the manifests (crd/bundle) regenerated if needed?
- [x] Did we add proper logging messages for operator actions?
- [x] Did we ensure compatibility with the previous version or cluster upgrade process?
- [x] Does the change support oldest and newest supported PXC version?
- [x] Does the change support oldest and newest supported Kubernetes version?

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/auto-tuning-8-0.log) |
| cross-site-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/demand-backup-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/pitr-8-0.log) |
| pitr-gap-errors-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/pitr-gap-errors-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/scaling-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/946ff49f/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1517/commits/946ff49fb20b5d5b475ab7e34562ba85445c001a
image: `perconalab/percona-xtradb-cluster-operator:PR-1517-946ff49f`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/auto-tuning-8-0.log) |
| cross-site-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/demand-backup-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/pitr-8-0.log) |
| pitr-gap-errors-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/pitr-gap-errors-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/scaling-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/3278c6b9/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1517/commits/3278c6b949ff8b4738538192f9a0161b24685e7b
image: `perconalab/percona-xtradb-cluster-operator:PR-1517-3278c6b9`

File: pkg/controller/pxc/service_test.go [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/percona/percona-xtradb-cluster-operator/pkg/pxc"
+)
+
+var _ = Describe("Service Labels Annotations Test", Ordered, func() {
```
Comment (inelpandzic) [inline pkg/controller/pxc/service_test.go:18]: You can name it only like `Service labels and annotations`.

**Review by inelpandzic — APPROVED** [review]

**Review by hors — CHANGES_REQUESTED** [review]

File: pkg/controller/pxc/service_test.go (hunk 1 above)
Comment (pooknull) [reply pkg/controller/pxc/service_test.go]: https://github.com/percona/percona-xtradb-cluster-operator/pull/1517/commits/64789d32c0277ee4d8717000206978deb28f3a71

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/auto-tuning-8-0.log) |
| cross-site-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/demand-backup-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/limits-8-0.log) |
| monitoring-2-0-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/one-pod-8-0.log) |
| pitr-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/pitr-8-0.log) |
| pitr-gap-errors-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/pitr-gap-errors-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/scaling-8-0.log) |
| scheduled-backup-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/scheduled-backup-8-0.log) |
| security-context-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1517/ef7a8052/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1517/commits/ef7a8052ef7778a309350c5e433092ce1437243e
image: `perconalab/percona-xtradb-cluster-operator:PR-1517-ef7a8052`

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**